README.md

```diff
@@ -53,21 +55,17 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(1). To validate a path, you only need memory for 2 pointers traversing from either ends of the path to the path's midpoint regardless of the number of tiles in the path.
```

Comment:
The parser takes its input one token at a time. It doesn't know where the end is
until it gets there. So the mechanical parsing process is to get the next token,
push it on a stack, and then compare the rest of the input. As a result, the
stack can grow to at least n/2, so the memory consumption will be O(n)

# -3